pkg/imgpkg/bundle/images_refs.go

```diff
@@ -17,11 +17,42 @@ import (
 
 type ImageRef struct {
 	lockconfig.ImageRef
-	IsBundle *bool
+	IsBundle  *bool
+	Copiable  bool
+	ImageType ImageType
+}
+
+// ImageType defines the type of Image
+type ImageType string
```

Comment:
one thing to note is that clients of imgpkg cannot assume that this is a final list of enum values. not sure how to best "enforce" that.

Reply:
I added that information as part of the comment